telegram/_files/inputfile.py

```diff
@@ -82,39 +85,13 @@ def __init__(
         ):
```

Comment:
above this line (line 84), when can `obj.name` be an `int`?

Reply:
Apparently that's the case for `InputFile(subprocess.Popen(…).stdout)` - see `test_subprocess_pip` in `test_inputfile`